### DISCUSSION TITLE
How do I make a custom badge?

### DISCUSSION THREAD

**Discussion by otilor** [badges/shields#5834] [General] [answered]
please answer

**Comment by anubra266** [comment]
Make your own badges! (Quick example: https://img.shields.io/badge/left-right-f39f37)

**Reply by otilor** [reply]
Thanks a lot.

**Comment by calebcartwright** [comment, marked as answer]
There's several different ways to make your own badge, one of which is the static badge builder mentioned above.

The better place to view them all is:
https://shields.io/#your-badge

**Reply by otilor** [reply]
Thanks a lot, man.
